src/xmime_internal.hpp

```diff
@@ -16,6 +16,10 @@
 
 #include "nlohmann/json.hpp"
 
+#if defined(WIN32)
```

Comment:
Well the cling and clang framework produce a lot of warnings in a Visual Studio build, which is very uggly as you don't see your own warnings anymore.

The following lines will disable the warnings completly. I think this is fine as it only for the external library.